### ISSUE TITLE
Does GitLab have issue tracking?

### ISSUE THREAD

**Issue by dionazakis** [gokuze345/reloj-biom-trico#2]
We built on top of GitLab's existing issue-tracking functionality and leverage the power of GitLab issue labels by utilizing them as lists on a Kanban board. You can construct different views of your issue board  [sublimation](http://printerlake.com/best-sublimation-printer-for-small-business/) printer while maintaining the same filtering and sorting abilities you see across the issue tracker

**Comment by pooledorothyw** [comment]
Yes, GitLab has issue tracking functionality. Similar to GitHub and Jira, GitLab provides a built-in issue tracking system that allows teams to manage and track tasks,[ bugs](https://cryptomarketingguide.com/), feature requests, and other issues related to software development projects.
